.github/workflows/build-and-test.yml

```diff
@@ -367,22 +367,21 @@ jobs:
           yarn config set network-timeout 600000 -g
           yarn install
 
-      - name: Download mamba installer for distribution (MacOS)
+      - name: Download micromamba for distribution (MacOS)
         if: matrix.os == 'macos-13'
-        run: curl -fsSLo Miniforge3-MacOSX-x86_64.sh "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-x86_64.sh"
-
-      - name: Install mamba for MacOS
-        if: matrix.os == 'macos-13'
-        run: bash Miniforge3-MacOSX-x86_64.sh -b -p dist/miniforge3
-
-      - name: Download mamba installer for distribution (Windows)
-        if: matrix.os == 'windows-latest'
-        run: curl -fsSLo Miniforge3-Windows-x86_64.exe "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Windows-x86_64.exe"
+        run: |
+          curl -Ls https://micro.mamba.pm/api/micromamba/osx-64/latest | tar -xvj bin/micromamba
+          mv bin/micromamba dist/
+          ./dist/micromamba --help

```

Comment:
Is this just to make sure the command works, and if it doesn't we'll fail / exit here? If so, it might be nice to have a short comment here.

Reply:
Yes, just testing the executable. Which might not be necessary any more, but I think it's fine to leave in. Added a comment